packages/gatsby-theme-newrelic/README.md

```diff
@@ -558,6 +563,26 @@ const Documentation = () => (
 );
 ```
 
+### `CookieConsentDialog`
+
+A dialog box that pops up asking for cookie consent. This component renders at the bottom of the screen and provides options for accepting or denying cookies.
+
+```js
+import { CookieConsentDialog } from '@newrelic/gatsby-theme-newrelic';
+```
+
+**Example**
+
+```js
+const myLayout = () => (
```

Comment:
[nit] `Layout`

Figure it probably best to use React names to ensure we communicate that this likely goes in a layout _component_ (which are capitalized)